packages/react-dom/src/shared/DOMProperty.js

```diff
@@ -224,12 +224,8 @@ const properties = {};
 
 // A few React string attributes have a different name.
 // This is a mapping from React prop names to the attribute names.
-new Map([
-  ['acceptCharset', 'accept-charset'],
-  ['className', 'class'],
-  ['htmlFor', 'for'],
-  ['httpEquiv', 'http-equiv'],
-]).forEach((attributeName, name) => {
+['acceptCharset', 'className', 'htmlFor', 'httpEquiv'].forEach((name, i) => {
+  const attributeName = ['accept-charset', 'class', 'for', 'http-equiv'][i];
```

Comment:
might be clearer if it was an array of array pairs? Like maybe just the argument to the original map. Less likely to accidentally shift one array but not the other